### PR TITLE
Fix importing empty initializers

### PR DESCRIPTION
**Pull request by tungld**
With Release build mode,  compiling yolov4 failed due to the following error:
```
[libprotobuf FATAL /usr/local/include/google/protobuf/repeated_field.h:1323] CHECK failed: (index) < (current_size_): 
terminate called after throwing an instance of 'google::protobuf::FatalException'
```
It happened when importing ResizeOp whose inputs were empty initializers (tensor<0xf32>). Below is ResizeOp in yolov4:
Screen Shot 2022-01-31 at 17 25 50

This patch fixes the issue by importing empty initializers as NoneType.

Signed-off-by: Tung D. Le <tung@jp.ibm.com>

### CODE REVIEW

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

**Review by etiotto — COMMENTED** [review]
@tungld can you add a "lit" test to check the codegen for a Resize operator with an empty initializer?

File: src/Builder/FrontendDialectHelper.cpp [hunk 1]
```diff
@@ -138,6 +138,13 @@ mlir::Value InitializedTensorMapping::EmitInitializerForInputTensor(
   // Initializer for input.
   onnx::TensorProto initializer = GetInitializedTensor(name);
 
+  // Return none if the initializer is an empty tensor tensor<0xf32>.
```
Comment (etiotto) [inline src/Builder/FrontendDialectHelper.cpp:141]: [typo]: "an empty tensor tensor<0xf32>." --> "an empty tensor."

**Comment by tungld** [comment]
> @tungld can you add a "lit" test to check the codegen for a Resize operator with an empty initializer?

Unfortunately, we have not yet had an infrastructure for this. This is an issue about importing onnx model so input should be an onnx model. This issue [#218](https://github.com/onnx/onnx-mlir/issues/218) is for such an infrastructure.
